nixos/tests/mediamtx.nix

```diff
@@ -23,6 +23,8 @@ import ./make-test-python.nix ({ pkgs, lib, ...} :
           DynamicUser = true;
           Restart = "on-failure";
           RestartSec = "1s";
+          # restart service without going through failed/inactive state that confuses wait_for_unit
+          RestartMode = "direct";
```

Comment:
I'm curious why it seems to be expected that this service will need to restart, but I suppose it was already like that.

Reply:
It needs to restart because the rtsp stream broker in the middle doesn't have its sockets up immediately, so it will have a few false starts on connecting to the stream broker.

We could probably try harder, but shrug. This works.